qir/qsharp-target-tests/TargetTestScripts/Intrinsic/ApplyUncontrolledH.qs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace QATSample {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Measurement;
+
+    @EntryPoint()
+    operation Main(): Result
+    {
+      use qubit = Qubit();
+
+      ApplyUncontrolledH(qubit);
```

Comment:
All of these "Apply*" callables don't need to be tested. They should not be included explicitly here.